components/yaml.rst

```diff
@@ -440,6 +440,34 @@ By default, digit-only array keys are dumped as integers. You can use the
     $dumped = Yaml::dump([200 => 'foo'], 2, 4, Yaml::DUMP_NUMERIC_KEY_AS_STRING);
     // '200': foo
 
+Dumping Collection of Maps
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, the collection of maps uses a hyphen on a separate line as a delimiter.
```

Comment:
Isn’t the same also true when dumping a collection of sequences?

Reply:
Option do not affect dump of sequences. This [code](https://github.com/symfony/symfony/commit/ec3f12b43bda6bc14008435636bacd0acff42226#diff-ae1ca77d752278d29e77b6fd168273d64b68e8fc3b9da8de7cca8f4644a4a827R68) enables compact nested mappings only when dumped value is hash.